pip/commands/run.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+from pip.basecommand import Command, SUCCESS
+
+from pip._vendor import rwt
+
+
+class RunCommand(Command):
+    """Show help for commands"""
+    name = 'run'
+    usage = rwt.commands.help_doc
```

Comment:
Do you mean the ```rwt``` tests? So they test the pip integration as well? Then they definitely should be added to the pip test suite, rather than being external.

We might be talking at cross purposes, though. The other pip subcommands have their own pages (```docs/reference/pip_list.rst``` for example). I was expecting a ```pip_run.rst``` alongside these.